visual_mode/visual_panels.md

```diff
@@ -0,0 +1,66 @@
+# Visual Panels
+
+## Concept
+
+Visual Panels is characterized by the following core functionalities.
+
+1. Split Screen
+2. Display the multiple screens such as Symbols, Registers, Stack, and you can even define your own panels too.
```

Comment:
change to "Display multiple screens such as Symbols, Registers, Stack, as well as customed panels"